gourmet/exporters/exportManager.py

```diff
@@ -2,7 +2,8 @@
 from gourmet.plugin import ExporterPlugin
 import gourmet.gtk_extras.dialog_extras as de
 from gourmet.threadManager import get_thread_manager, get_thread_manager_gui
-from gi.repository.GLib import get_user_special_dir, USER_DIRECTORY_PICTURES
+from gi.repository.GLib import (get_user_special_dir, USER_DIRECTORY_PICTURES,
+    USER_DIRECTORY_DOCUMENTS)
```

Comment:
This variable looks unused, but the same is for the `USER_DIRECTIRY_PICTURES`, so I have vague feeling it's needed somewhere under the hood. Can you comment on this?

Reply:
Oh, I was completely wrong, I'm sorry. I found the place where we use this.